### PR TITLE
Feature/745 show save status

### LINKED ISSUE #745: UI - Saving is not clear on all pages that save
# Task
* Some sort of ui change is needed for saving/errors besides just notifications, it's not clear when an action has successfully saved or error'd

### PR DESCRIPTION
**Pull request by kalitopix**
# Problem
referenced issue(s):

# Solution

# Testing

closes #745 

Screen Shot 2019-10-25 at 9 53 38 AM

### CODE REVIEW

**Comment by kalitopix** [comment]
@rochaeb , @rssk 
Currently, I showed saving status before the save button and made this for pipeline and consortia save/update.

Are there any other places to use this?

File: packages/coinstac-ui/app/render/components/common/status-button-wrapper.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    return <i className="animated bounceIn fa fa-exclamation-circle" style={style} />
+  }
+
+  renderStatus = () => {
+    const { status } = this.props
+
+    switch(status) {
+      case 'pending':
+        return this.pendignIcon
```
Comment (rochaeb) [inline packages/coinstac-ui/app/render/components/common/status-button-wrapper.js]: typo

**Review by rochaeb — APPROVED** [review]

File: packages/coinstac-ui/app/render/components/common/status-button-wrapper.js (hunk 1 above)
Comment (kalitopix) [reply packages/coinstac-ui/app/render/components/common/status-button-wrapper.js]: @rochaeb , what do you mean?

**Comment by rochaeb** [comment]
yeah, I think only pipeline and consortia have this type of screen where this would fit

File: packages/coinstac-ui/app/render/components/common/status-button-wrapper.js (hunk 1 above)
Comment (rssk) [reply packages/coinstac-ui/app/render/components/common/status-button-wrapper.js]: pending vs pendign

**Review by rssk — APPROVED** [review]
Looks great!

**Pull request merged**